draft-ietf-tls-tls13.md

```diff
@@ -1,4 +1,4 @@
----
+/---
```

Comment:
The insertion of the / at the beginning is wrong.

On Mon, May 18, 2015 at 8:58 AM, seanturner <notifications@github.com>
wrote:

> In draft-ietf-tls-tls13.md
> <https://github.com/tlswg/tls13-spec/pull/152#discussion_r30520749>:
>
> > @@ -1,4 +1,4 @@
> > ----
> > +/---
>
> I can�t figure out which change this applies to. spt
> … <#14d67bfca4e6f40e_>
> On May 16, 2015, at 17:11, ekr <notifications@github.com> wrote: In
> draft-ietf-tls-tls13.md: > @@ -1,4 +1,4 @@ > ---- > +/--- This seems like
> it might be wrong. � Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tlswg/tls13-spec/pull/152/files#r30520749>.
>